### ISSUE TITLE
How I Can Display PointCloud Map?

### ISSUE THREAD

**Issue by heshiwen1989** [uber/xviz#612]
Hello, 
     I Want Use This Tool Display PointCloud Map, But This Map has Too Many Point(~16M), So I Can Not Send This Map EveryTime: For Example The PointCloud Map Like This: 
![图片](https://user-images.githubusercontent.com/28886748/90355845-2803e100-e080-11ea-9bd9-75d0602da7a1.png)

**Comment by pai-qi** [comment]
> Hello,
> I Want Use This Tool Display PointCloud Map, But This Map has Too Many Point(~16M), So I Can Not Send This Map EveryTime: For Example The PointCloud Map Like This:
> ![2020-08-17 11-52-07屏幕截图](https://user-images.githubusercontent.com/28886748/90355905-5681bc00-e080-11ea-83da-695c421e125d.png)

Hello, Have you solved this problem yet？

**Comment by heshiwen1989** [comment]
> > Hello,
> > I Want Use This Tool Display PointCloud Map, But This Map has Too Many Point(~16M), So I Can Not Send This Map EveryTime: For Example The PointCloud Map Like This:
> > ![2020-08-17 11-52-07屏幕截图](https://user-images.githubusercontent.com/28886748/90355905-5681bc00-e080-11ea-83da-695c421e125d.png)
> 
> Hello, Have you solved this problem yet？

no, because the web side not buffer this map, if you not send everytime, then this map will not display
